### ISSUE TITLE
Feature request: threshold for regressions

### ISSUE THREAD

**Issue by astrofrog** [spacetelescope/asv#439]
The webpage to see all regressions is very useful, but it would be nice to have a way to control the threshold for regressions - e.g. I want to be able to sort by date and see only things that got worse by say a factor of 2.

**Comment by pv** [comment]
Non-interactive: gh-456. (Interactive one could also be nice, but a noninteractive one is needed in order to apply filtering to the regression feed.)

**Comment by astrofrog** [comment]
👍
